spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -458,6 +459,81 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
     SerializationUtils.clone(this)
   }
 
+  def clone(deepCopy : Boolean): AbstractModule[A, B, T] = {
```

Comment:
name of this method is conflict with java Object.clone()

Reply:
I removed the default the value ,  it should be ok now , previously `def clone(deepCopy : Boolean = true):`

Reply:
Sorry, I did not notice. It should work.